examples/fileio_features_xc/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(fileio_features_xc)
+
+# sandbox
+set(XMOS_SANDBOX_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+message(STATUS "XMOS_SANDBOX_DIR: ${XMOS_SANDBOX_DIR}")
```

Comment:
Is this debug, or printed for a particular reason? What's special about this variable when you don't print any of the others? This is also in the other CMakeLists.txt files.